.github/workflows/sanitycheck.yml

```diff
@@ -29,3 +29,21 @@ jobs:
 
     - name: detect non-ASCII encoding and trailing space
       run: python3 ./build/sanitycheck.py
+
+  check-format:
```

Comment:
This one would probably worth its own workflow or get merged with other compilation/stylecop workflow.
Putting it here means the job would get triggered even for a PR that only touched Markdown files.

Not a blocker though.

Reply:
just updated